spec/schemas/flow_framework._common.yaml

```diff
@@ -0,0 +1,100 @@
+openapi: 3.1.0
+info:
+  title: Schemas of flow_framework._common category
+  description: Schemas of flow_framework._common category
+  version: 1.0.0
+paths: {}
+components:
+  schemas:
+    FlowFrameworkException:
```

Comment:
Does this one inherit from an OpenSearch error type?

Reply:
yes , it inherit from OpenSearchException

`FlowFrameworkException extends OpenSearchException`

Reply:
That makes it an Exception, though.  "Error" in exception handling has a specific meaning (the program should be halted).

Reply:
https://github.com/opensearch-project/flow-framework/blob/b3f9d6514d2c0530c8bffac654fe175452673169/src/main/java/org/opensearch/flowframework/rest/RestCreateWorkflowAction.java#L113C5-L146C55
How about these, should i call it error or exception ?

Reply:
It's interesting, the rest response includes the word "error" as an "error message" although it's an "exception".  I'll defer to @dblock here on what the appropriate spec/api terminology is here.

Reply:
🤷‍♂️ @nhtruong ?

Reply:
We've been calling them errors through out the Spec (i.e. ReponseError). The clients also consider non 2XX responses as errors for error handling. As far as Client-Server communication is concerned (i.e. Web API), there's no distinction between Error and Exception, in my opinion. Any non-2XX response tells the user that something has gone wrong, and we usually call it a 500 Error Code etc instead of a 500 exception code.